voicevox_engine/synthesis_engine.py

```diff
@@ -64,6 +64,7 @@ def __init__(
         yukarin_s_forwarder,
         yukarin_sa_forwarder,
         decode_forwarder,
+        speakers,
```

Comment:
list型と間違えやすそうな名前なので、型ヒント書くと優しいかもです。
```suggestion
        speakers: str,
```